modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -34,8 +35,17 @@ def submit
           evss_data = evss_mapper_service(auto_claim).map_claim
           evss_service.submit(auto_claim, evss_data)
 
-          generate_526_pdf(pdf_data)
-
+          pdf_string = generate_526_pdf(pdf_data)
+          file_name = "#{SecureRandom.hex}.pdf"
+          path = ::Common::FileHelpers.generate_temp_file(pdf_string, file_name)
```

Comment:
it should all be stored in the model, so we shouldn't have to worry about the naming tooooo much. we could make the argument of what to name it, but I think I'd be fine with whatever.

Reply:
also @mchristiansonVA I'm assuming this path variable is a temporary placeholder because the pdf_generator is awaiting changes?

Reply:
Yeah I did some testing using StringIO and handling the pdf in-memory instead of temporarily writing the file to /temp, but it seemed like ActionDispatch::Http::UploadedFile wanted a real File specified for "tempfile". Open to suggestions if you think there's a better way to handle this.